test-data/unit/fixtures/fine_grained.pyi

```diff
@@ -20,7 +23,7 @@ class str:
 
 class float: pass
 class bytes: pass
-class tuple: pass
+class tuple(Generic[T]): pass
 class function: pass
 class ellipsis: pass
-class list: pass
```

Comment:
It looks like there is still one more ``tuple`` not fixed in ``isinstancelist.pyi``.